src/pages/knowledge-base.tsx

```diff
@@ -304,9 +304,9 @@ const DocumentList: FunctionComponent<DocumentListProps> = (
                   })}
                 </Box>
                 <Typography variant="caption" display="block">
-                  {`${
-                    node.frontmatter.updatedDate !==
-                    node.frontmatter.publishDate
+                  {`${node.frontmatter.updatedDate && 
+                    (node.frontmatter.updatedDate !==
+                    node.frontmatter.publishDate) 
```

Comment:
The other two implementations of this don't have `(...` the parens around the comparison.  Could we keep that consistent, whichever way you prefer?

Reply:
thanks for the catch - done